specs/derivation.md

```diff
@@ -517,23 +517,29 @@ New frames for timed-out channels are dropped instead of buffered.
 
 The channel-bank can only output data from the first opened channel.
 
-Upon reading, first all timed-out channels are dropped.
+Upon reading, first all expired channels are removed from the known complete channels.
+
+A channel is expired if: `current_l1_block.number > channel.open_l1_block.number + max(SEQUENCING_WINDOW_SIZE, CHANNEL_TIMEOUT)`
+
```

Comment:
I would expect that `SEQUENCING_WINDOW_SIZE` is always larger than `CHANNEL_TIMEOUT` but I haven't seen anywhere that explicitly requires it, hence the max.